test_driver/user_open_app.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:heist/main.dart';
+
+Future main() async {
+  // This line enables the extension
+  enableFlutterDriverExtension();
+
+  // Call the `main()` of your app or call `runApp` with whatever widget
+  // you are interested in testing.
+  FirebaseOptions options = new FirebaseOptions(
```

Comment:
Maybe we can store these locally for now, until we set up an android emulator on Travis, in which case we can encrypt them as environment variables in the build.